src/view-models/result-record.test.ts

```diff
@@ -203,6 +207,82 @@ describe("ResultRecord", () => {
 
     // #endregion errorCount
 
+    // -----------------------------------------------------------------------------------------
+    // #region doesNotHaveErrors
+    // -----------------------------------------------------------------------------------------
+
+    describe("doesNotHaveErrors", () => {
+        test("when errors is null, returns true", () => {
+            expect(
+                ResultRecordFactory.build({
+                    errors: (null as unknown) as any[],
+                }).doesNotHaveErrors()
+            ).toBeTrue();
+        });
+
+        test("when errors is undefined, returns true", () => {
+            expect(
+                ResultRecordFactory.build({
+                    errors: undefined,
+                }).doesNotHaveErrors()
+            ).toBeTrue();
+        });
+
+        test("when errors is empty array, returns true", () => {
+            expect(
+                ResultRecordFactory.build({ errors: [] }).doesNotHaveErrors()
+            ).toBeTrue();
+        });
+
+        test("when errors is not empty, returns false", () => {
+            // Arrange
+            const errors = ResultErrorRecordFactory.buildList(1);
+
+            // Act & Assert
+            expect(
+                ResultRecordFactory.build({
+                    errors: errors,
+                }).doesNotHaveErrors()
+            ).toBeFalse();
+        });
+    }); // end doesNotHaveErrors
+
+    // -----------------------------------------------------------------------------------------
+    // #region getErrorMessageFor
+    // -----------------------------------------------------------------------------------------
+
+    describe("#getErrorMessageFor", () => {
+        it("When no error exists for given key, then returns undefined", () => {
+            // Arrange
+            const errorKey = "TEST_ERROR_KEY";
+            const resultRecord = ResultRecordFactory.build({
```

Comment:
do these need to be statically referenced? I think we usually usually do `Factory.build<ResultRecord>(...)`

Reply:
I can make this change to be consistent with everywhere else.

Reply:
I can make this change to be consistent with everywhere else.